cassandra/io/asyncioreactor.py

```diff
@@ -176,7 +176,10 @@ def push(self, data):
             )
         else:
             # avoid races/hangs by just scheduling this, not using threadsafe
```

Comment:
Please move the comment to `create_task` line.

Reply:
moved it